src/System.Text.Encoding/tests/NegativeEncodingTests.cs

```diff
@@ -36,11 +36,19 @@ public static void GetByteCount_Invalid(Encoding encoding)
             Assert.Throws<ArgumentOutOfRangeException>("chars", () => encoding.GetByteCount(new char[3], 2, 2));
             Assert.Throws<ArgumentOutOfRangeException>("chars", () => encoding.GetByteCount(new char[3], 3, 1));
             Assert.Throws<ArgumentOutOfRangeException>("chars", () => encoding.GetByteCount(new char[3], 4, 0));
+
+            char[] chars = new char[3];
+            fixed (char* pChars = chars)
+            {
+                char* pCharsLocal = pChars;
```

Comment:
right, I forgot we cannot use the fixed variables inside the lambda expressions or anonymous methods.